core/src/main/scala/org/http4s/headers/`Cross-Origin-Embedder-Policy`.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import cats.parse.Parser
+import cats.parse.Parser.string
+import org.typelevel.ci._
+
+sealed abstract class `Cross-Origin-Embedder-Policy`(val value: String)
```

Comment:
Would it be possible to add some Scala-Docs with a reference (link) to the RFC, or equivalent authoritative document, where this Header is specified?